src/controller/editingcontroller.js

```diff
@@ -69,6 +69,13 @@ export default class EditingController {
 		const selection = doc.selection;
 		const markers = this.model.markers;
 
+		// When taking over an editing view, disable the rendering until the first
+		// changes were made to the model. This allows a free manipulation of the view root
+		// attributes without any risk of breaking anything, for instance, the initial content
+		// of the DOM root element. If the rendering was allowed, any change of attributes would
+		// trigger rendering and synchronize the DOM root with a (still) empty model.
+		this.view._renderingDisabled = true;
```

Comment:
Unfortunately, it does not help much. It is enough to have any model change, even a one which does not change anything in the model document (like [data parsing](https://github.com/ckeditor/ckeditor5-engine/blob/05e4e200a378f99d94afbe03d84baf6b2170165a/src/controller/datacontroller.js#L329)) to cause change block which [sets this flag to `true` and runs rendering](https://github.com/ckeditor/ckeditor5-engine/blob/v12.0.0/src/controller/editingcontroller.js#L83-L84). However, @ma2ciek is [working on it](https://github.com/ckeditor/ckeditor5-engine/issues/1653).

Reply:
I reverted those changes in favor of #1661.